doc/packages/emacs.section.md

```diff
@@ -97,9 +113,13 @@ You can install it like any other packages via `nix-env -iA myEmacs`. However, t
 }
 ```
 
-This provides a fairly full Emacs start file. It will load in addition to the user's personal config. You can always disable it by passing `-q` to the Emacs command.
+This provides a fairly full Emacs start file. It will be loaded in addition to the user's personal config. Further, it can be disabled at will: Emacs accepts the command-line argument `--no-init-file` (short form `-q`) to not load the init files.
```

Comment:
> it can be disabled at will  ... `-q`) to not load the init files

This is a bit confusing.

Presumably, it refers to `default.el`.  However, `-q` makes Emacs not load both `default.el` and `init.el`.

We should first mention that `inhibit-default-init` in `init.el` can prevents `default.el` from being loaded.  Then we mention `-q` make both of them not be loaded.